sdk/nodejs/acm/certificateValidation.ts

```diff
@@ -27,10 +29,10 @@ import * as utilities from "../utilities";
  *     domainName: "example.com",
  *     validationMethod: "DNS",
  * });
- * const zone = pulumi.output(aws.route53.getZone({
+ * const zone = aws.route53.getZone({
```

Comment:
@pgavlin, are these changes expected? Are they from the newer `tf2pulumi`?

Reply:
Yes.